DEPRECATED.md

```diff
@@ -8,6 +8,8 @@ A logged warning is expected for each deprecated item that is in deprecation win
 
 ## Version 1.9.0 (pending)
 
+* Use of the boolean field `bugfix_reverse_encode_order` in [http_connection_manager.proto](https://github.com/envoyproxy/envoy/blob/master/api/envoy/config/filter/network/http_connection_manager/v2/http_connection_manager.proto) is deprecated. This is a bug fix to resolve issue [#4599](https://github.com/envoyproxy/envoy/issues/4599) for envoy where it is designed to have the encode filters in reversed order to the configuration.
```

Comment:
Can you point out at the beginning of this bullet point something to the effect of:

"Order of execution of the encoder filter chain has been reversed. Prior to this release cycle it was incorrect, see `#4599`. In the 1.9.0 release cycle we introduced `bugfix_reverse_encode_order` to temporarily support both old and new behaviors."

Then the rest can be pretty much what you have there..

Reply:
Done.